src/main/java/org/interledger/btp/ErrorPacket.java

```diff
@@ -0,0 +1,16 @@
+package org.interledger.btp;
+
+import org.immutables.value.Value.Immutable;
+
+import java.time.Instant;
+
+@Immutable
+public interface ErrorPacket extends BtpPacketBase {
+
+  BtpErrorType getError();
```

Comment:
Not a blocker, but I think Checkstyle will `warn` when Javadoc is missing and we should probably document each method in each interface.

Reply:
CheckStyle issues fixed